app/src/main/res/values/dimens.xml

```diff
@@ -23,7 +23,17 @@
     <dimen name="cbs_text_view_title_margin">6dp</dimen>
     <dimen name="launcher_title_text_size">16sp</dimen>
     <dimen name="btns_text_view_title_margin">4dp</dimen>
-    <dimen name="child_program_linear_layout_margin">8dp</dimen>
     <dimen name="launcher_btn_height">80dp</dimen>
     <dimen name="launcher_btn_width">0dp</dimen>
+    <dimen name="checkbox_margin_top">8dp</dimen>
```

Comment:
Duplicate

Reply:
Can you please elaborate more, what is duplicate here?

Reply:
8dp is already defined, use that